Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/ConverterConfig.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+#pragma warning disable CA2235 // Mark all non-serializable fields
+    [Serializable]
+    [System.Diagnostics.DebuggerDisplay("{converterName}, enabled={enabled}")]
+    public struct ConverterConfig
+    {
+        public bool enabled;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)